open_academy/models/session.py

```diff
@@ -20,3 +20,20 @@ def _compute_taken_seats(self):
             if record.number_of_seats > 0:
                 taken_seats = round((len(record.attendees_ids) * 100) / record.number_of_seats)
             record.taken_seats = taken_seats
+
+    @api.onchange('number_of_seats','attendees_ids')
+    def _onchange_invalid_values(self):
+        if self.number_of_seats < 0:
+            return {
+                'warning': {
+                'title': "Something bad happened",
+                'message': "Number of seats must be a number and must be greater than 0",
```

Comment:
Why specifying that this needs to be a number? 

The field itself is an integer, when trying to save it will trigger a warning like this

<img width="763" alt="Screen Shot 2022-08-26 at 9 32 58 a m" src="https://user-images.githubusercontent.com/47955967/186927994-2acef544-fff9-4b86-adaf-b1764c547a4d.png">

Reply:
But is impossible to add a string in that field due to the warning that appears automatically, why would be necessary to validate that it is not an `str`?

Reply:
Your error message states that

> "Number of seats must be a number and must be greater than 0",

That's why I said this:

> Why specifying that this needs to be a number?
> The field itself is an integer, when trying to save it will trigger a warning

In short, change the message if no strings can enter the onchange, since this is already validated by the ORM

Reply:
Ok!, I got it, something like "Number of seats must be greater than 0" would be better?